inginious/frontend/plugins/problem_bank/pages/api/search_task_api.py

```diff
@@ -0,0 +1,32 @@
+from inginious.frontend.plugins.utils.admin_api import AdminApi
+
+
+class SearchTaskApi(AdminApi):
```

Comment:
Maybe another name is prefered here because we have a ```Filter task API``` and a ```SearchAPI``` maybe this can be called. ```AvailableTasksApi```?